src/apiv2.ts

```diff
@@ -374,7 +371,7 @@ export class Client {
       reqTimeout = setTimeout(() => {
         controller.abort();
       }, options.timeout);
-      fetchOptions.signal = controller.signal;
+      fetchOptions.signal = controller.signal as RequestInit["signal"];
```

Comment:
this is due to node-fetch AbortController types not being up-to-date with the runtime AbortController which supports more features